doc/data/messages/d/duplicate-value/details.rst

```diff
@@ -0,0 +1,2 @@
+This error message is emitted when a set contains the same value two or more times.
```

Comment:
Information like this should probably included in the message description in its declaration.

Reply:
Only need to move this to `pylint/checkers/base/basic_checker.py`